catatonit.c

```diff
@@ -482,6 +482,11 @@ int main(int argc, char **argv)
 	if (argc < 1 && !run_as_pause)
 		bail_usage("missing program name");
 
+	char *kill_pgid_env = secure_getenv("CATATONIT_KILLPG");
+	if (kill_pgid_env != NULL) {
+		kill_pgid = true;
+	}
```

Comment:
Please use the Linux kernel style:

```suggestion
	if (kill_pgid_env)
		kill_pgid = true;
```

or even just do:

```suggestion
	kill_pgid |= getenv("CATATONIT_KILLPG") != NULL;
```